File: 백준/Bronze/10951. A＋B － 4/A＋B － 4.py
import sys

while True:
    line = sys.stdin.readline().strip()
    if not line:
        break

    A, B = map(int, line.split())
    if A < 0 or B < 0 or A > 10 or B > 10:
        break
    else:
        sys.stdout.write(str(A + B) + "\n")

sys.stdout.flush()  # 출력 버퍼 비우기

File: 백준/Bronze/10951. A＋B － 4/A＋B － 4.java
import java.io.*;
import java.util.*;

public class Main {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
		do {
			String line = br.readLine();
			if (line == null || line.isEmpty()) {
				break;
			}

			String[] input = line.split(" ");

			int A = Integer.parseInt(input[0]);
			int B = Integer.parseInt(input[1]);

			if (A < 0 || B < 0 || A > 10 || B > 10) {
				break;
			} else {
				bw.write((A + B) + "\n");
			}

		} while (true);

		bw.flush();
		bw.close();
		br.close();
	}
}